### ISSUE TITLE
Check duration of one loop

### ISSUE THREAD

**Issue by PcTim** [powtac/Autarc_LAN_User_Stats#53]
How much time needs a 255.255.255.0 loop?
Maybe we have to improve our code to decrease this time. e.g. use only 2 pings, instead of 4.

**Comment by powtac** [comment]
Loops should be done very fast anyway. Because when a device does not respond, it does not respond or is broken or whatever. We don't want to store data about broken or very slow devices...

**Comment by powtac** [comment]
Lops must get much faster! currently it takes 31 minutes for a 0/255 scan! With post to server every 20 results and pingretries = 1.

**Comment by PcTim** [comment]
You are right. One ping should be enough. If we remove/deactivate some debugging informations we could save a lot of time. Also we can improve the whole functions. I have to adapt the functions to our new paradigm. Now we have a few unused parameters...

**Comment by PcTim** [comment]
Also #63 should help

**Comment by PcTim** [comment]
We also have to look how much longer it took to store the log on SD. Potentially we can improve this code.

**Comment by PcTim** [comment]
cb59d498c2a282b2f19f109c948b6e3fddc8a08a is really improving our scanning duration
